docs/source/user_guide/data_checks.ipynb

```diff
@@ -454,6 +455,41 @@
     "    print(\"Error:\", message['message'])"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Datetime Format\n",
+    "\n",
+    "Datetime information is a necessary component of time series problems, but sometimes the data we deal with may contain flaws that make it impossible for time series models to work with them. For example, in order to identify a frequency in the datetime information there has to be equal interval spacing between data points i.e. January 1, 2021, January 2, 2021, January 3, 2021, ...etc. If instead there are random jumps in the datetime data i.e. January 1, 2021, January 2, 2021, January 5, 2021, then a frequency can't be inferred. Another common issue with datetime information is if it isn't properly sorted. Datetime values that aren't monotonically increasing or decreasing will encounter the same problem.\n",
```

Comment:
"i.e. January 1, 2021, January 2, 2021, January 3, 2021," --> can you explain that these dates are equally one day apart?

Reply:
Some suggestions on clarity:

Another common issue with datetime information is if it isn't properly sorted. --> TS models also cannot handle datatime info that isn't properly sorted. 

Datetime values that aren't monotonically increasing or decreasing will encounter the same problem. --> Frequency also cannot be inferred from datetime values that are not monotonically increasing or decreasing?